src/subprocess-posix.cc

```diff
@@ -24,6 +24,13 @@
 #include <sys/wait.h>
 #include <spawn.h>
 
+#ifdef __FreeBSD__
+#  include <sys/param.h>
+#  if defined USE_PPOLL && __FreeBSD_version < 1002000
```

Comment:
fwiw we don't put spaces between # and the word after it in ninja; maybe you can zap them in a follow-up